assets/css/custom.css

```diff
@@ -289,20 +296,21 @@ input[type="text"].letters-input {
 .swatches {
   height: 100%;
   display: flex;
-  flex-wrap: wrap;
   align-items: center;
   justify-content: center;
+  width: 100%;
 }
 
 .swatch {
-  height: 90%;
   text-align: center;
+  max-height: 60%;
```

Comment:
hm, seems like this could result in stretched/distorted images in combination with `width: 100%`